lib/Controller/DocumentController.php

```diff
@@ -355,7 +355,7 @@ public function create($mimetype,
 		}
 
 		if (!$content){
-			$content = file_get_contents(dirname(__DIR__) . self::ODT_TEMPLATE_PATH);
+			$content = file_get_contents(dirname(dirname(__DIR__)) . self::ODT_TEMPLATE_PATH);
```

Comment:
one dirname call should be enought :wink:

Reply:
or so you want to call dirname on the whole string including the template path, then there is an issue with the parentheses.

Reply:
No. The thing is that dirname on dir gives us `<path>/lib` we need the dirname of that to get to the root of `<path>`

Reply:
Ah, now I get it. :+1:

Reply:
Then approve please :wink: